src/parsedmarc/utils.py

```diff
@@ -121,7 +121,7 @@ def query_dns(
     domain = str(domain).lower()
     record_type = record_type.upper()
     cache_key = f"{domain}_{record_type}"
-    if cache:
+    if cache is not None:
```

Comment:
Just to clarify: are we doing the comparison with `None` because the empty cache is falsey?

Reply:
The answer is yes - this is a backport of https://github.com/domainaware/parsedmarc/pull/494